README.md

```diff
@@ -119,6 +119,7 @@ Inline always open version
 | calendar-button               | Boolean         | false       | Show an icon that that can be clicked    |
 | calendar-button-icon          | String          |             | Use icon for button (ex: fa fa-calendar) |
 | calendar-button-icon-content  | String          |             | Use for material-icons (ex: event)       |
+| dayCellContent                | Function        | day.date    | Use to render custom content in day cell |
```

Comment:
@charliekassel No, actually the the default value is a Function that returns a Number - I just added "day.date" instead of "day => day.date" to the default column for a lack of space. Will fix this by not adding the default value to the doc, as it's probably more confusing as it is now.